src/reducers/index.js

```diff
@@ -0,0 +1,85 @@
+import { combineReducers } from 'redux';
```

Comment:
Will consider using the immutable library in future - wanted to avoid the added testing complexity for now.  Using updateObject method to enforce immutability